src/main/java/com/google/flourbot/entity/EntityModuleImplementation.java

```diff
@@ -55,80 +56,86 @@ public EntityModuleImplementation(DataStorage dataStorage) {
   }
 
   private Optional<Trigger> getTrigger(Map<String, Object> triggerData) {
+    try {
+      String triggerType = (String) triggerData.get("type");
 
-    String triggerType = (String) triggerData.get("type");
+      switch (triggerType) {
+        case ("Command Trigger"):
+          String command = (String) triggerData.get("command");
+          Trigger trigger = new CommandTrigger(command);
 
-    switch (triggerType) {
-      case ("Command Trigger"):
-        String command = (String) triggerData.get("command");
-        Trigger trigger = new CommandTrigger(command);
+          return Optional.of(trigger);
 
-        return Optional.of(trigger);
-
-      default:
-        return Optional.empty();
+        default:
+          return Optional.empty();
+        }
+    } catch (NullPointerException e) {
+      return Optional.empty();
     }
   }
 
   private Optional<Action> getAction(Map<String, Object> actionData) {
-
-    String actionType = (String) actionData.get("type");
-
-    switch (actionType) {
-      case ("Sheet Action"):
-        ArrayList<String> columnStringList = (ArrayList<String>) actionData.get("columnValue");
-        // Converts into ENUM type
-        ArrayList<SheetEntryType> columnTypeList = new ArrayList<>();
-        for (String type : columnStringList) {
+    try {
+      String actionType = (String) actionData.get("type");
+
+      switch (actionType) {
+        case ("Sheet Action"):
+          ArrayList<String> columnStringList = (ArrayList<String>) actionData.get("columnValue");
+          // Converts into ENUM type
+          ArrayList<SheetEntryType> columnTypeList = new ArrayList<>();
+          for (String type : columnStringList) {
```

Comment:
nit: I assume this is just formatting clean-up - these should be submitted separately to avoid confusion.

Reply:
I was adding the indents for a try catch for NullPointerException. Hmm, It seems like git recognized it as adding and deleting new lines

Reply:
I will also keep in mind to reformat in a separate branch in the future!